binding/Binding/SkiaApi.cs

```diff
@@ -1908,24 +1908,24 @@ internal static unsafe class SkiaApi
 		[DllImport (SKIA, CallingConvention = CallingConvention.Cdecl)]
 		public extern static sk_textblob_t sk_textblob_builder_make (sk_textblob_builder_t builder);
 		[DllImport (SKIA, CallingConvention = CallingConvention.Cdecl)]
-		public extern static void sk_textblob_builder_alloc_run_text (sk_textblob_builder_t builder, sk_paint_t font, int count, float x, float y, int textByteCount, sk_string_t lang, SKRect* bounds, out SKTextBlobBuilderRunBuffer runbuffer);
+		public extern static void sk_textblob_builder_alloc_run_text (sk_textblob_builder_t builder, sk_paint_t font, int count, float x, float y, int textByteCount, sk_string_t lang, SKRect* bounds, out SKRunBufferInternal runbuffer);
 		[DllImport (SKIA, CallingConvention = CallingConvention.Cdecl)]
-		public extern static void sk_textblob_builder_alloc_run_text_pos_h (sk_textblob_builder_t builder, sk_paint_t font, int count, float y, int textByteCount, sk_string_t lang, SKRect* bounds, out SKTextBlobBuilderRunBuffer runbuffer);
+		public extern static void sk_textblob_builder_alloc_run_text_pos_h (sk_textblob_builder_t builder, sk_paint_t font, int count, float y, int textByteCount, sk_string_t lang, SKRect* bounds, out SKRunBufferInternal runbuffer);
 		[DllImport (SKIA, CallingConvention = CallingConvention.Cdecl)]
-		public extern static void sk_textblob_builder_alloc_run_text_pos (sk_textblob_builder_t builder, sk_paint_t font, int count, int textByteCount, sk_string_t lang, SKRect* bounds, out SKTextBlobBuilderRunBuffer runbuffer);
+		public extern static void sk_textblob_builder_alloc_run_text_pos (sk_textblob_builder_t builder, sk_paint_t font, int count, int textByteCount, sk_string_t lang, SKRect* bounds, out SKRunBufferInternal runbuffer);
 
 
 		// text blob builder run buffer
 		[DllImport (SKIA, CallingConvention = CallingConvention.Cdecl)]
-		public extern static void sk_textblob_builder_runbuffer_set_glyphs (ref SKTextBlobBuilderRunBuffer buffer, UInt16* glyphs, int count);
+		public extern static void sk_textblob_builder_runbuffer_set_glyphs (ref SKRunBufferInternal buffer, UInt16* glyphs, int count);
```

Comment:
Do we still need these?

Reply:
Yes. We still need to allocate the memory on the native side.

Reply:
Skia already allocates the memory with `SkiaApi.sk_textblob_builder_alloc_run_text` etc what do I miss here?

Reply:
Ah, my bad. This is the set.

Just noticed that the manual copy might have performance implications https://github.com/mono/SkiaSharp/pull/865/files#r296629605

Reply:
We can probably remove these.

Reply:
Removed these in favor of the new Span<T>.CopyTo with better memory operations